src/IMangrove.sol

```diff
@@ -40,16 +39,37 @@ interface IMangrove {
     uint id,
     uint prev
   );
-  event OrderComplete(
+  // tried to add all relevant data for an order.
+  // same
+  event OrderComplete( // this could be deduced from the OfferSuccess events emtitted from "execute" in Mangrove
     address indexed outbound_tkn,
     address indexed inbound_tkn,
     address indexed taker,
+    uint takerWants,
+    uint takerGives,
     uint takerGot,
     uint takerGave,
+    bool fillWants,
+    uint penalty,
```

Comment:
we may want to call this `bounty` or `penaltyReceived`

Reply:
Had already changed it to bounty in MgvLib, missed it in IMangrove.